tests/network/vmi_networking.go

```diff
@@ -838,6 +838,17 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				}, timeout, time.Second).Should(Succeed(), fmt.Sprintf("migration should succeed after %d s", timeout))
 			}
 
+			commandDhClient := func(cmd string) error {
```

Comment:
Can this be a regular function?

But maybe you can check if `console.RunCommand` can be used instead.
(you can see usage of it in `PingFromVMConsole` for example)